Podcast/FeedViewController.swift

```diff
@@ -85,6 +86,9 @@ class FeedViewController: ViewController, UITableViewDelegate, UITableViewDataSo
         refreshControl.endRefreshing()
     }
 
+}
+
+extension FeedViewController: UITableViewDataSource, UITableViewDelegate {
```

Comment:
are we just putting this as an extension for clarity sake or is there a specific reason?

Reply:
Well clarity, but also the FeedViewController declaration line was getting too long

Reply:
lol .. ok with me but make a comment that its for clarities sake so that prior iOS devs don't get confused that there was a reason